pynamodb/expressions/condition.py

```diff
@@ -74,12 +108,12 @@ def __bool__(self):
 class Between(Condition):
     format_string = '{path} {operator} {0} AND {1}'
 
-    def __init__(self, attribute, value1, value2):
-        super(Between, self).__init__(attribute, 'BETWEEN', value1, value2)
+    def __init__(self, path, lower, upper):
+        super(Between, self).__init__(path, BETWEEN, lower, upper)
 
 
 class BeginsWith(Condition):
     format_string = '{operator} ({path}, {0})'
 
-    def __init__(self, attribute, prefix):
-        super(BeginsWith, self).__init__(attribute, 'begins_with', prefix)
+    def __init__(self, path, prefix):
+        super(BeginsWith, self).__init__(path, 'begins_with', prefix)
```

Comment:
the current `BEGINS_WITH` const is for the old api which is capitalized and not lowercase :(